recipe/meta.yaml

```diff
@@ -8,10 +8,13 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/linbox-team/linbox/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 47f025fe7aab204fb276307f32a562de357c887c6ef589b9e4f7a57a3b58ecaf
+  sha256: 7b69ab617b15ca6684b29d08531ed8899cd4d07ec75799e5d624c4c0a60dc6b2
+  patches:
+    # An upstream patch that Sage uses for 8.2
+    - patches/9e6e5dd-f4b483c.patch
```

Comment:
Do you have a link for the patch?

Reply:
Ok.

Reply:
So it's not from upstream? Was it sent upstream by sage developers?

Reply:
It's originally from Sage but it is already in upstream master in the exact same form.